e3db/types/note_options.py

```diff
@@ -0,0 +1,81 @@
+from datetime import date
+
+# Look for a possible alternative to a NoteOptions class. Not a huge priority at the moment, 
+# but this may not be what is wanted. 
+class NoteOptions():
+    
+    def __init__(self, client_id: str, max_views: int, note_name: str, expiration: date, # Check `date` type for expiration. 
```

Comment:
Regarding date type

https://github.com/tozny/e3db-clients-go/blob/master/storageClient/api.go#L29

It should be an RFC339 string
https://datatracker.ietf.org/doc/html/rfc3339

Reply:
forcing a user to create an RFC339 feels wrong. I would think this should take a date (or an epoch milli or second, as python dates are a struggle with timezones) and then at serialization time we can handle the conversion